src/pyairios/registers.py

```diff
@@ -121,6 +131,38 @@ def encode(self, value: T) -> list[int]:
         raise AiriosInvalidArgumentException(f"Unsupported type {type(value)}")
 
 
+class U8Register(NumberRegister[int]):
```

Comment:
This just add range checking. Please see an alternative implementation which avoids code dup:

https://github.com/scabrero/pyairios/commit/a08f567d362227aca6185dc95fa2dbb9e4704c46
https://github.com/scabrero/pyairios/commit/fbab9cff5c865d600bc70e22296f98636852f455